### PR TITLE
Toolkit 200.4.0 Release items

### PR DESCRIPTION
**Pull request by mhdostal**
This PR preps the Toolkit for the v200.4.0 release. It includes:

- Update version numbers for all apps (Examples, AuthenticationExample, JobManagerExample) to 200.4.0
- Updates the copyright notice in the main README

This PR should be tested after the SDK release and then merged just prior to [merging](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/669) `v.next` into `main` and creating the TK release.

### CODE REVIEW

File: Sources/ArcGISToolkit/Documentation.docc/Resources/FeatureFormView/FeatureFormView.png
Comment (dfeinzimer) [inline Sources/ArcGISToolkit/Documentation.docc/Resources/FeatureFormView/FeatureFormView.png:1]: If we add a second file suffixed with "~dark" it'll be used used for dark mode users:

demo

So `FeatureFormView.png` and `FeatureFormView~dark.png`. I went ahead and re-created both variants for you to download and use:
https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/2b1fd137-1f7b-46f5-b69d-76a63922ff8c
https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/8606d001-12be-497e-933e-78444c226720

**Comment by mhdostal** [comment]
I moved the FeatureForm stuff out of this PR and into a separate PR than can be merged separately.

**Review by dfeinzimer — COMMENTED** [review]
One other thing. Should we append 2024 onto the end of the top level README here? Or do something like "2022-2024"?

Screenshot 2024-04-04 at 1 00 57 PM

**Comment by mhdostal** [comment]
@dfeinzimer Good call on the copyright stuff. I made the change.

**Review by dfeinzimer — APPROVED** [review]

File: Sources/ArcGISToolkit/Documentation.docc/Resources/FeatureFormView/FeatureFormView.png
Comment (mhdostal) [reply Sources/ArcGISToolkit/Documentation.docc/Resources/FeatureFormView/FeatureFormView.png:1]: That's perfect, thanks!!

**Pull request merged**